srcpkgs/zeroc-mcpp/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'zeroc-mcpp'
+pkgname=zeroc-mcpp
+version=2.7.2.14
+revision=1
+wrksrc="mcpp-${version}"
+build_style=gnu-makefile
+short_desc="Simplified fork of mcpp from ZeroC, for building ICE"
+maintainer="a dinosaur <nick@a-dinosaur.com>"
+license="BSD-2-Clause"
+homepage="https://github.com/zeroc-ice/mcpp"
+distfiles="https://github.com/zeroc-ice/mcpp/archive/v${version}.tar.gz"
+checksum=1a426cf2d513ffd66634384d5445dcce1aac83be1789a56044013eab8ca8ffff
+nostrip=yes
+conflicts=mcpp-devel
+
+post_patch() {
+	vsed 's|LIBDIR = lib64|LIBDIR = lib|' -i Makefile
+}
```

Comment:
You can do `make_install_args="LIBDIR=lib"` instead.

Reply:
Ty for fast review🚀
I'll apply these when the checks are finished.

Reply:
I remember why I did it this way now, the Makefile has no DESTDIR so it tries to install in /usr/lib and ends up in the masterdir's /usr instead.

Reply:
I've tried patching the Makefile instead to make a real install rule, I've been careful about it in case it can be upstreamed.
Btw LIBDIR has to be set for both `make` and `make install` because LIBDIR is used to pick the subdir to place the binary ..ya idk.